install/kubernetes/README.md

```diff
@@ -1,13 +1,15 @@
 # Install Istio on an existing Kubernetes cluster
 
-Please follow the installation instructions from [istio.io](https://istio.io/docs/tasks/installing-istio.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/quick-start.html).
 
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
 * istio.yaml - use this file for installation without authentication enabled
 * istio-auth.yaml - use this file for installation with authentication enabled
-* istio-cluster-wide.yaml - use this file for installation cluster-wide with authentication enabled
+* istio-one-namespace.yaml - use this file for installation without authentication enabled and CA only watch one namespace
```

Comment:
put a period after CA and start a new sentence "Istio control plane and app in the same watched namespace."

Reply:
Done